src/RiakClient/Models/Index/RiakStreamedIndexResult.cs

```diff
@@ -24,17 +24,21 @@ namespace RiakClient.Models.Index
     using Extensions;
     using Messages;
 
+    /// <summary>
+    /// Represents a result to a streaming index query.
+    /// </summary>
     public class RiakStreamedIndexResult : IRiakIndexResult
     {
         private readonly IEnumerable<RiakResult<RpbIndexResp>> responseReader;
         private readonly bool includeTerms;
 
-        public RiakStreamedIndexResult(bool includeTerms, IEnumerable<RiakResult<RpbIndexResp>> responseReader)
+        internal RiakStreamedIndexResult(bool includeTerms, IEnumerable<RiakResult<RpbIndexResp>> responseReader)
```

Comment:
Aha I missed that it was just the ctor. Can the ctors of the other result classes be changed to `internal` too?

Does making this `internal` affect someone who'd like to mock out things when using our lib?